src/Microsoft.OData.Core/ODataMetadataFormat.cs

```diff
@@ -128,6 +164,23 @@ public override string ToString()
             throw new ODataException(Strings.General_InternalError(InternalErrorCodes.ODataMetadataFormat_CreateOutputContextAsync));
         }
 
+        private static bool IsJsonMetadata(ODataMediaType contentType)
+        {
+            // by default, it's XML metadata
+            if (contentType == null)
+            {
+                return false;
+            }

```

Comment:
you could put contentType!=null as part of the check in 175, but I'm okay if you want to leave as is for readability.

Reply:
Yes, put it as it for readability 

---
In reply to: [518471298](https://github.com/OData/odata.net/pull/1913#discussion_r518471298) [](ancestors = 518471298)